connector/semconvcheckerconnector/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+type: semconvchecker
+scope_name: otelcol/semconvchecker
```

Comment:
the scope name will automatically be generated as the package name, this configuration not needed

```suggestion
```